package.json

```diff
@@ -15,7 +15,7 @@
 		"build-website": "npm run parse-readme-template && cd website && npm ci --legacy-peer-deps && npm run build",
 		"serve-website": "cd website && npm run serve",
 		"watch": "npx chokidar \"*.ts\" \"taqueria-utils/**/*.ts\" \"taqueria-protocol/**/*.ts\" -c \"npm run build || npx play-sound beep.mp3\" --initial",
-		"build-and-watch-vscode": "npx chokidar \"*.ts\" \"taqueria-vscode-extension/**/*.ts\" \"taqueria-protocol/**/*.ts\" -c \"npm run compile -w taqueria-vscode-extension || npx play-sound beep.mp3\" --initial",
+		"build-and-watch-vscode": "npx chokidar \"*.ts\" \"taqueria-vscode-extension/**/*.ts\" \"taqueria-protocol/**/*.ts\" -c \"time npm run compile -w taqueria-vscode-extension || npx play-sound beep.mp3\" --initial",
```

Comment:
Was `time` for debugging and meant to be deleted?

Reply:
I actually wanted to keep it. But now I think it might cause problems. I'll delete it.